include/libsemigroups/containers.hpp

```diff
@@ -25,6 +25,7 @@
 #include <type_traits>  // for is_default_constructible
 #include <vector>       // for vector, allocator
 
+#include "adapters.hpp"             // for LIBSEMIGROUPS_ASSERT
```

Comment:
Presumably not included for `LIBSEMIGORUPS_ASSERT` really?

Reply:
It does seem a bit unlikely doesn't it :thinking: